src/views/Proposal.vue

```diff
@@ -49,7 +49,7 @@
             >
               {{ _shorten(choice, 32) }}
               <a
-                v-if="_get(payload, `metadata.plugins.aragon.choice${i + 1}`)"
+                v-if="payload.metadata.plugins?.aragon.choice.i + 1"
```

Comment:
Hum, does this work `choice.i + 1`? I think it should be like this
```suggestion
                v-if="payload.metadata.plugins?.aragon?.choice?.[i + 1]"
```

Reply:
Yea, I wasn't to sure about my solution